nx-dev/nx-dev/redirect-rules.js

```diff
@@ -461,7 +461,7 @@ const nxCloudUrls = {
   '/ci/recipes/on-premise/auth-saml':
     '/ci/recipes/enterprise/on-premise/auth-saml',
   '/ci/recipes/on-premise/auth-saml-managed':
-    '/ci/recipes/enterprise/on-premise/auth-saml-managed',
```

Comment:
Please add also redirection from `/ci/recipes/enterprise/on-premise/auth-saml-managed` to `/ci/recipes/enterprise/on-premise/auth-saml`. Same for the above